src/pages/LandingPage/landingpage.css

```diff
@@ -0,0 +1,45 @@
+
+.body-section{
+    display: flex;
+    grid-column: 20rem 1fr;
```

Comment:
I believe property name is grid-template-columns

Reply:
Thanks 
I'll  fix it.